### ISSUE TITLE
Default Boolean types in avro schema are not translated into correct Boolean type in python pydantic code generation

### ISSUE THREAD

**Issue by DaanRademaker** [godatadriven/pydantic-avro#19]
Hello!
Thanks for this library. I would like to report an issue with the generation of booleans for default values I generate Pydantic models from an asvc file. 

for example the following avro schema 
```json
{
  "name": "test",
  "type": "boolean",
  "default": false
}
```

would lead to this in the pydantic model generated

```python
test: bool = false
```

Lowercase false is not a correct boolean in Python of course so, something is going wrong in the mapping.

**Comment by DaanRademaker** [comment]
I think I might have made a good fix for it. I will send a merge request soon.

**Comment by DaanRademaker** [comment]
I created a fix for this in pull request: 
https://github.com/godatadriven/pydantic-avro/pull/20

**Comment by ffinfo** [comment]
will take a look, thanks for the contribution

**Comment by ffinfo** [comment]
fixed in #20
